### ISSUE TITLE
New keyword Element Attribute Should Be

### ISSUE THREAD

**Issue by manycoding** [robotframework/SeleniumLibrary#531]
I need to check user created attributes and it'd nice if I have this keyword:
    Element Attribute Should Be    //input[contains(@id, "id")]@aria-disabled     ${data.prerequisites}

Instead of:
    ${value}=    Get Element Attribute    //input[contains(@id, "id")]@aria-disabled
    Should Be Equal    ${value}    ${data.prerequisites}

**Comment by aaltat** [comment]
I think this could be useful. I think that one should define the attribute which they are checking, example something like:
`|Element Attribute Should Be | xpath=//input | key=value |`

As a workaround one can do this with `Page Should Contain` keyword and xpath. Example:
`| Page Should Contain Element | xpath=//input[@id="I'd" and @aria-disabled="True"] |` But having own keyword for this makes it more clearer what the keyword is actually testing, because xpath can come quite complex.

Would you like to make a pull request to implement this?

**Comment by manycoding** [comment]
Forgot to mention the attribute itself.
As for the pull request - I'll think about it.

**Comment by emanlove** [comment]
Just a word of encouragement to you @manycoding as it is always great to get more contributors. If you need any help just ask.

**Comment by drobota** [comment]
https://github.com/robotframework/Selenium2Library/pull/570

**Comment by php-coder** [comment]
> https://github.com/robotframework/Selenium2Library/pull/570

Here is the working link: #570

**Comment by aaltat** [comment]
That is because project changed name and direct links with old name got broken.
